### ISSUE TITLE
Files attached to assets to actually be deleted

### ISSUE THREAD

**Issue by maciejgrabowy** [grokability/snipe-it#7674]
**Server (please complete the following information):**
 - Snipe-IT Version 4.8.0
 - OS: Debian 10
 - Web Server: Apache

**Is your feature request related to a problem? Please describe.**
Files attached to assets (and other things too, I'm assuming) via the file tab don't actually get deleted from the server after pressing the "delete" button and can be found in /storage/private_uploads/assets. They are also included in backups, which leads to significant bloat and potentially confidential documents sitting there, forever. The purge option doesn't include them as well.
Unless I'm missing something, of course, and there's something wrong on my end.

**Describe the solution you'd like**
An option for the files to be physically deleted would be very welcome, since manually removing them every time isn't a solution in the long run.

**Comment by snipe** [comment]
The files should definitely be getting deleted. Is there anything in your error logs?

**Comment by maciejgrabowy** [comment]
No, nothing gets logged.

**Comment by maciejgrabowy** [comment]
Well, things are still not being deleted, so yes, it's relevant. Any ideas?

**Comment by maciejgrabowy** [comment]
Annoyingly relevant, yes.

**Comment by maciejgrabowy** [comment]
Yup.

**Comment by maciejgrabowy** [comment]
Hi, friend bot. Yes, they're still not being deleted. If only someone could point me in the right direction...

**Comment by misilot** [comment]
Yes, still relevant. This applies to both local storage and at least s3 storage.

**Comment by misilot** [comment]
Also still applies to v5.1.5

**Comment by snipe** [comment]
I believe this was fixed a long time ago.
